tutorials/2-05-external_NACA_RBFcurving/parameter_hopr.ini

```diff
@@ -0,0 +1,55 @@
+!=============================================================================== !
+! OUTPUT
+!=============================================================================== !
+projectname    = NACA0012_RBFCurv   ! name of the project (used for filenames)
+Debugvisu      = T                      ! Write debug mesh to tecplot file
+outputFormat   = 1
+DebugvisuLevel = 2                      ! Level 0: visualization of linear mesh and BC (default), 
+                                      ! Level 1: + curved surface visualization (_SplineSurf.*), if useCurveds
+                                      ! Level 2: + curved volume visualization (_SplineVol.*), if useCurveds
+!=============================================================================== !
+! MESH
+!=============================================================================== !
+Mode             = 3                    ! 1 Cartesian 3 CGNS 4 STAR-CD V3
+bugfix_ANSA_CGNS = T
+nZones           = 2                    ! number of zones
+filename         = naca0012_omg.cgns    ! name of mesh file
+
+meshscale        = 0.001                ! scales all input meshes by a factor
+spacequandt      = 1000.                ! tolerance for mesh connectivity (multiplicates with real tolerance)
+!=============================================================================== !
+! CURVED
+!=============================================================================== !
+useCurveds         = T
+curvingMethod      = 3
+SplitElemFile      = naca0012_omg_split.cgns
+dosortijk          = T
+BoundaryOrder      = 5
+checkElemJacobians = T                  ! checks the Jacobian and scaled Jacobian for each element
+useRBF             = T
+SupportRadius      = 0.01 ! Chosse in the same ballpark as the length of the elements
```

Comment:
Typo and formatting

Reply:
Typo changed and formatting cleaned up. Please resolve is sufficient